osquery/tables/system/windows/logical_drives.cpp

```diff
@@ -0,0 +1,84 @@
+#include "osquery/core/windows/wmi.h"
+#include <osquery/tables.h>
+
+namespace osquery {
+namespace tables {
+
+QueryData genLogicalDrives(QueryContext& context) {
+  Row r;
+  QueryData results;
+
+  WmiRequest wmiLogicalDiskReq(
+      "select DeviceID, DriveType, FreeSpace, Size, FileSystem from "
+      "Win32_LogicalDisk");
+  std::vector<WmiResultItem>& wmiResults = wmiLogicalDiskReq.results();
+  for (unsigned int i = 0; i < wmiResults.size(); ++i) {
+    unsigned int driveType = 0;
+    std::string deviceId;
+    wmiResults[i].GetString("DeviceID", deviceId);
+    r["device_id"] = deviceId;
+    wmiResults[i].GetUnsignedInt32("DriveType", driveType);
+    wmiResults[i].GetString("FreeSpace", r["free_space"]);
+    wmiResults[i].GetString("Size", r["size"]);
+    wmiResults[i].GetString("FileSystem", r["file_system"]);
+
+    switch (driveType) {
+    case 0:
+      r["type"] = TEXT("Unknown");
+      break;
+    case 1:
+      r["type"] = TEXT("No Root Directory");
+      break;
+    case 2:
+      r["type"] = TEXT("Removable Disk");
+      break;
+    case 3:
+      r["type"] = TEXT("Local Disk");
+      break;
+    case 4:
+      r["type"] = TEXT("Network Drive");
+      break;
+    case 5:
+      r["type"] = TEXT("Compact Disc");
+      break;
+    case 6:
+      r["type"] = TEXT("RAM Disk");
+      break;
+    }
+
+    bool bootPartition = false;
+
+    std::stringstream assoc_query_ss;
```

Comment:
There's a little bit of inconsistency between `snake_case` and `camlCase` throughout your code. We typically prefer `camlCase`ing for Windows cpp - would you mind going through and unifying all of your local vars?